hoodie-client/src/main/java/com/uber/hoodie/config/HoodieStorageConfig.java

```diff
@@ -34,6 +34,12 @@
   public static final String DEFAULT_PARQUET_BLOCK_SIZE_BYTES = DEFAULT_PARQUET_FILE_MAX_BYTES;
   public static final String PARQUET_PAGE_SIZE_BYTES = "hoodie.parquet.page.size";
   public static final String DEFAULT_PARQUET_PAGE_SIZE_BYTES = String.valueOf(1 * 1024 * 1024);
+  // used to size log files
+  public static final String LOGFILE_SIZE_MAX_BYTES = "hoodie.logfile.max.size";
```

Comment:
please add this to configuration section of docs as well, since its pretty important

Reply:
done